dedupe/ckan_api.py

```diff
@@ -116,21 +117,31 @@ def get_duplicate_identifiers(self, organization_name, is_collection):
         if is_collection:
             filter_query = '%s AND collection_package_id:*' % filter_query
 
-        response = self.get('/3/action/package_search', params={
-            'fq': filter_query,
-            'facet.field': '["identifier"]',
-            'facet.limit': -1,
-            'facet.mincount': 2,
-            'rows': 0,
-            })
+        if self.identifier_type == 'identifier':
+            response = self.get('/3/action/package_search', params={
+                'fq': filter_query,
+                'facet.field': '["identifier"]',
+                'facet.limit': -1,
+                'facet.mincount': 2,
+                'rows': 0,
+                })
+        else: 
+            response = self.get('/3/action/package_search', params={
+                'fq': filter_query,
+                'facet.field': '["guid"]',
+                'facet.limit': -1,
+                'facet.mincount': 2,
+                'rows': 0,
+                })
+
```

Comment:
You should just be able to inject the identifier type, something like `'facet.field': '["' + self.identifier_type + '"]'`. Then we can remove the if/else.